flaskbb/auth/views.py

```diff
@@ -401,25 +354,15 @@ def registration_service_factory():
             registration_service_factory=registration_service_factory
         )
     )
+
     register_view(
         auth,
         routes=['/reset-password'],
-        view_func=ForgotPassword.as_view('forgot_password')
-    )
-
-    def reset_service_factory():
```

Comment:
hehe cool that you got rid of that

Reply:
It's still around, just moved to auth.services.factories. I ended up needing the activation service in the post-register hook so I migrated all three there instead of having duplicating it.